gatsby-node.js

```diff
@@ -113,3 +133,15 @@ exports.createPages = ({ graphql, actions }) => {
     );
   });
 };
+
+exports.onCreateNode = ({ node, actions }) => {
```

Comment:
What triggers this "onCreateNode"?

Reply:
onCreateNode will be called whenever Gatsby creates or updates a node of its internal Redux store.